Aquality.Selenium.Template/Aquality.Selenium.Template/Utilities/CustomImageComparator.cs

```diff
@@ -4,37 +4,40 @@
 
 namespace Aquality.Selenium.Template.Utilities
 {
-    public static class CustomImageComparator
+    public class CustomImageComparator
     {
-        private static IVisualizationConfiguration visualizationConfiguration;
+        private readonly ImageComparator imageComparator;
 
-        public static void Init(float defaultThreshold = 0.1F, int comparisonWidth = 20, int comparisonHeight = 20)
+        public CustomImageComparator(float defaultThreshold, int comparisonWidth, int comparisonHeight)
         {
-            visualizationConfiguration = new VisualizationConfiguration(defaultThreshold, comparisonWidth, comparisonHeight);
+            var visualizationConfiguration = new VisualizationConfiguration(defaultThreshold, comparisonWidth, comparisonHeight);
+            imageComparator = new ImageComparator(visualizationConfiguration);
         }
 
-        public static float Compare(Image firstImage, Image secondImage, float? threshold = null)
+        public float Compare(Image firstImage, Image secondImage, float? threshold = null)
         {
-            var imageComparator = new ImageComparator(visualizationConfiguration);
             return imageComparator.PercentageDifference(firstImage, secondImage, threshold);
         }
+    }
 
-        private class VisualizationConfiguration : IVisualizationConfiguration
-        {
-            public float DefaultThreshold { get; }
+    public class VisualizationConfiguration : IVisualizationConfiguration
```

Comment:
public class should be located in the different file. Please change back to private or move to a new file

Reply:
Fixed